amalgamate.py

```diff
@@ -1,31 +1,34 @@
 #!/usr/bin/env python
+"""
+Amalgamate PyNE's C++ library sources into a single header and source file.
+
+This script consolidates selected C++ components of the PyNE library into:
+    - pyne.h   : Self-contained header
+    - pyne.cpp : Self-contained source
 
-"""Amalgate pyne C++ library sources into a single source and a single header file.
-This makes the C++ portion of pyne more portable to other projects.
+Inspired by the JsonCpp amalgamation tool:
+    http://svn.code.sf.net/p/jsoncpp/code/trunk/jsoncpp/amalgamate.py
 
-Originally inspired by JsonCpp: http://svn.code.sf.net/p/jsoncpp/code/trunk/jsoncpp/amalgamate.py
+Usage:
+    python amalgamate.py [-s OUTPUT.cpp] [-i OUTPUT.h] [-f file1.h file2.cpp ...] [-o OUTPUT_DIR]
 """
+
 from __future__ import print_function, unicode_literals
 import os
-import sys
-import io
+import subprocess
 from argparse import ArgumentParser
 
-sys.path.append('pyne')
-import pyne_version as pv
-
-pv.write_cpp_header('src')
+# Configuration
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This is a good improvement, but if/when we change the location of `amalgamate.py`, we'll either have to update each location that this is used with some relative path changes, or this will be updated once with some relative path changes.

The latter seems preferred, in which case, this variable may be better called `BASE_DIR` since we want it to always point to the root of the source file system